src/main/js/helpers/axis.js

```diff
@@ -329,35 +428,41 @@ const getAverageTicksCount = (rangeY, rangeY2) => {
  * @param {number} lowerLimit - Lower limit of the Y or Y2 Axis
  * @param {number} upperLimit - Upper limit of the Y or Y2 Axis
  * @param {number} ticksCount - Number of ticks between the upper and lower limits
+ * @param {boolean} allowCalibration - Whether this property is true or false
  * @returns {(Array)} returns array of values to be used as tick labels
  */
 const generateYAxesTickValues = (
     lowerLimit,
     upperLimit,
-    ticksCount = constants.DEFAULT_TICKSCOUNT
+    ticksCount = constants.DEFAULT_TICKSCOUNT,
+    allowCalibration = true
```

Comment:
Why we are setting `allowCalibration` value to true by default?

Reply:
Because allowCalibration is true by default according to the documentation